development/docker-compose.yml

```diff
@@ -18,10 +21,8 @@ services:
       WORDPRESS_DB_PASSWORD: "${DB_ROOT_PASSWORD}"
     depends_on:
       - db
-
-    links:
-      - db
-
+    networks:
+      - backend
```

Comment:
We should probably move the network-related changes to a separate task/pull request.

Reply:
this was handled in #103